data_validation/query_builder/random_row_builder.py

```diff
@@ -12,15 +12,64 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import random
 import ibis
-from typing import List
+import ibis.expr.operations as ops
+import ibis.expr.types as tz
+import ibis.expr.rules as rlz
+import ibis.backends.base_sqlalchemy.compiler as sql_compiler
+from ibis_bigquery import BigQueryClient
+from ibis.backends.pandas.client import PandasClient
+import ibis.backends.pandas.execution.util as pandas_util
 
+from ibis.expr.signature import Argument as Arg
+from typing import List
 from data_validation import clients
+from io import StringIO
 
 
 """ The QueryBuilder for retreiving random row values to filter against."""
 
 
+######################################
+### Adding new datasources should be
+### done by adding the Client and
+### syntax for random sorts in the
+### dict below. If upstream injection
+### is required, feel free to reach
+### out to dhercher
+######################################
+RANDOM_SORT_SUPPORTS = {
+    PandasClient: "NA",
+    BigQueryClient: "RAND()",
```

Comment:
```suggestion
    BigQueryClient: "RAND()",
    ImpalaClient: "RAND()", 
```

Hive uses the RAND() function the same way BQ does. We will also need to import ImpalaClient for this.

Reply:
This should be added in a separate PR and tested, so removing from here